docs/index.html

```diff
@@ -1,86 +1,79 @@
-            menuId: "operating-system",
-          },
-          sync: true,
-          theme: "classic",
-        },
-        tabs: {
-          theme: "material",
-        },
-        search: {
-          paths: "auto",
-          placeholder: {
-            "/": "Type to search",
-          },
-          noData: {
-            "/": "No Results",
-          },
-        },
-        plugins: [
-          EditOnGithubPlugin.create(
-            "https://github.com/asdf-vm/asdf/blob/master/docs/",
-            {
-              customURLs: {
-                "https://raw.githubusercontent.com/asdf-vm/asdf-plugins/master/README.md":
-                  "https://github.com/asdf-vm/asdf-plugins/edit/master/README.md",
-                "https://raw.githubusercontent.com/asdf-vm/asdf/master/CHANGELOG.md":
-                  "https://github.com/asdf-vm/asdf/edit/master/CHANGELOG.md",
-              },
-            }
-          ),
-        ],
-      };
+        window.$docsify = {
+            name: "asdf",
+            repo: "https://github.com/asdf-vm/asdf",
+            el: "#app",
+            coverpage: ['/', '/pt-br/'],
+            loadNavbar: true,
+            loadSidebar: true,
+            notFoundPage: {
+                '/': '_404.md',
+                '/pt-br/': '/pt-br/_404.md'
+            },
+            subMaxLevel: 2,
+            auto2top: true,
+            alias: {
+                ".*?/plugins-all": "https://raw.githubusercontent.com/asdf-vm/asdf-plugins/master/README.md",
+                ".*?/changelog": "https://raw.githubusercontent.com/asdf-vm/asdf/master/CHANGELOG.md",
+            },
+            markdown: {
+                smartypants: true,
+            },
+            themeColor: "#AA00D4",
+            formatUpdated: "{MM}/{DD} {HH}:{mm}",
+            select: {
+                detectOperatingSystem: {
+                    enabled: true,
+                    menuId: "operating-system",
+                },
+                sync: true,
+                theme: "classic",
+            },
+            tabs: {
+                theme: "material",
+            },
+            search: {
+                paths: "auto",
+                placeholder: {
+                    "/": "Type to search",
+                },
+                noData: {
+                    "/": "No Results",
+                },
+            },
+            plugins: [
+                EditOnGithubPlugin.create(
+                    "https://github.com/asdf-vm/asdf/blob/master/docs/", {
+                        customURLs: {
+                            "https://raw.githubusercontent.com/asdf-vm/asdf-plugins/master/README.md": "https://github.com/asdf-vm/asdf-plugins/edit/master/README.md",
+                            "https://raw.githubusercontent.com/asdf-vm/asdf/master/CHANGELOG.md": "https://github.com/asdf-vm/asdf/edit/master/CHANGELOG.md",
+                        },
+                    }
+                ),
+            ],
+        };
```

Comment:
Why were these lines changed? I think I prefer to the two space indentation.

Reply:
I making fixes in format in this file.
please review files
thanks for marking the errors